lxd/instance/drivers/driver_qemu.go

```diff
@@ -1039,6 +1039,49 @@ func (d *qemu) saveState(monitor *qmp.Monitor) error {
 	return nil
 }
 
+// validateRootDiskStatefulStop validates the state of the root disk before stopping the instance.
```

Comment:
This comment explains *what* the function does, which is fine, but a reader can work that out by looking at the code. 

But much more important is the *why* it does what it does. And here we've now lost the context of this function as the useful comment has been left in one of the call sites.

> // The "size.state" of the instance root disk device must be larger than the instance memory.
> // Otherwise, there will not be enough disk space to write the instance state to disk during any subsequent stops.
> // (Only check when migration.stateful is true, otherwise the memory won't be dumped when this instance stops).

Lets integrate this comment into the function's comment and remove from the call site.